features/ui/button/index.ts

```diff
@@ -1 +1 @@
-export { Button } from "./button";
+export { Button, ButtonContainer } from "./button";
```

Comment:
If you remove the old `Button` implementation from this file you also don't have the awkwardness of these two components. As an outsider I would be surprised that I should use `ButtonContainer` if I want to render a button